netbox/extras/models/scripts.py

```diff
@@ -76,6 +77,14 @@ def _get_name(cls):
 
         return scripts
 
+    def clean(self):
```

Comment:
This should probably go on the ManagedFile class. We can tweak the method to reference the inheriting class itself so that there's no overlap between scripts and reports, for instance.